### PR TITLE
Staking/misc refactors

### PR DESCRIPTION
**Pull request by holygits**
Screen Shot 2021-02-22 at 14 04 42

- Refactor mystake into individual react components 
- Make manage stake a modal
- Update staking thingy on landing page

TODO:
- [x] Ensure manage stake operations which require controller use the controller account or show some warning etc.
- [x] fix lint + build

### CODE REVIEW

File: packages/app-staking/src/ManageStake/index.tsx [hunk 1]
```diff
@@ -58,11 +57,9 @@ function ManageStake ({ className, isVisible }: Props): React.ReactElement<Props
     useEffect((): void => {
       if (method) {
         const methodName = api.findCall(method.callIndex).method;
-        if (methodName === 'nominate' && accountIdVec.length === 0 || stakedAccountId === null) {
```
Comment (holygits) [inline packages/app-staking/src/ManageStake/index.tsx:61]: will revert this

**Comment by KarishmaBothara** [comment]
The UI looks cool.. One thing we would need to change is the pop up that appears on On-boarding/ new stake page, we would need to update the link for manage stake to my stake page... Also, do we need to have a help page on my stake page too? for user to know how to use manage stake modal?

The one issue I see is for Alice stash why the stake amount on the modal appears 0, I feel it should be 100.. for other bonding it appears fine..I think its fine.. only happens on local node... Nikau is good..
Screen Shot 2021-02-22 at 2 25 25 PM

**Comment by holygits** [comment]
Thanks for the feedback. had the same thought re help message on manage stake modal.
I decided to remove the manage stake link altogether

File: packages/react-components/src/StakingInputExtrinsic/options/method.tsx [hunk 2]
```diff
@@ -8,7 +8,7 @@ import React from 'react';
 import { ApiPromise } from '@polkadot/api';
 
 export default function createOptions (api: ApiPromise, sectionName: string): DropdownOptions {
-  let section = api.tx[sectionName];
+  let section = {...api.tx[sectionName]};
```
Comment (KarishmaBothara) [inline packages/react-components/src/StakingInputExtrinsic/options/method.tsx:11]: donno how but this was missing in this branch, but available in manage_stake_page branch at -https://github.com/cennznet/ui/blob/93bc2d0d7085cd08a3460a4a99c31aaca805fff7/packages/react-components/src/StakingInputExtrinsic/options/method.tsx#L11

**Pull request merged**